app/config/config.yml

```diff
@@ -17,3 +14,21 @@ chameleon_system_cms_text_field:
 chameleon_system_java_script_minification:
   js_minifier_to_use: "jshrink"
 
+doctrine:
+  dbal:
+    host:           '%database_host%'
+    port:           '%database_port%'
+    dbname:         '%database_name%'
+    user:           '%database_user%'
+    password:       '%database_password%'
+    driver:         'pdo_mysql'
+    server_version: '5.7'
+    charset:        'utf8'
+    default_table_options:
+      charset: 'utf8'
+      collate: 'utf8_unicode_ci'
+    options:
+      1006: 1
```

Comment:
What advantages has compression (by default)?

Reply:
Good catch, this wasn't intended.